Xwt.Gtk/Xwt.GtkBackend/MenuBackend.cs

```diff
@@ -70,6 +70,23 @@ public void InitializeBackend (object frontend, ApplicationContext context)
 				return bar;
 			}
 		}
+
+		Pango.FontDescription customFont;
+
+		public virtual object Font {
+			get {
+				return customFont ?? menu.Style.FontDescription;
+			}
+			set {
+				customFont = (Pango.FontDescription) value;
+				foreach (var item in menu.AllChildren) {
+					var bin = item as Gtk.Bin;
```

Comment:
Shouldn't this be `Gtk.Container`? Gtk.Bin is a specialized single-child container.